style.css

```diff
@@ -0,0 +1,226 @@
+body{
+  background: black;
+}
+
+
+/*Contact Us Page*/
+
+
+section {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+
+.soc img{
+  width: 25px;
+}
+
+section::before
+{
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 50%;
+  height: 100%;
+  background: #ee122a;
+
+}
+
+section .container{
+  position: relative;
+  min-width: 1100px;
+  min-height: 550px;
+  display: flex;
+}
+
+section .container .contactinfo{
+  position: absolute;
+  top: 40px;
+  width: 350px;
+  height: calc(100% - 80px);
+  background: black;
+  z-index: 1;
+  padding: 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  box-shadow: 0 20px 20px rgba(0, 0, 0, 0.2);
```

Comment:
group similar properties, to know more about how to group you can check any senior members project from their repositories.

Reply:
Similar Properties grouped. Kindly review.